plugins/entando-plugin-jacms/src/main/java/org/entando/entando/plugins/jacms/apsadmin/content/bulk/bulk-actions.xml

```diff
@@ -166,7 +166,7 @@
 	
     <package name="jacms_do/jacms/Content/Category" namespace="/do/jacms/Content/Category" extends="jacms_do/jacms/Content/BulkBase">
 
-		<action name="entry" class="jacmsContentCategoryBulkAction" method="entry" >
+		<action name="entry*" class="jacmsContentCategoryBulkAction" method="entry" >
```

Comment:
Want to make sure this change is intentional. A greedy match on the action name seems aggressive here. Is this needed?

Reply:
Yes, it's intentional. The open/close action of the tree nodes (on request) requires intercepting actions that start with "entry" in that namespace.
Perhaps it is better to put a less generic action name.

Reply:
@paddeo Lets make that change. That name match is too broad.